pages/api/restricted.js

```diff
@@ -0,0 +1,17 @@
+import { getServerSession } from "next-auth/next"
+import { authOptions } from "./auth/[...nextauth]"
+
+export default async (req, res) => {
```

Comment:
write a name of the function. If there are more than one default async then there will be conflicts in future

Reply:
this is from the next-auth documentation sir